test/unit/services/rest/index.spec.js

```diff
@@ -69,6 +69,13 @@ describe('Rest', () => {
         Rest.validateRestOptions(restOptions);
       }).to.throw();
     });
+
+    it('should convert unsecure parameter to secure if the secure parameter is not set', () => {
+      restOptions.unsecure = false;
```

Comment:
I think it would be better to set this to `true` and validate that `restOptions.secure` is `false`, otherwise we might test that it was defaulting `secure` to `true` correctly